goosebit/settings/schema.py

```diff
@@ -24,6 +25,17 @@ def get_json_permissions(self):
         return [str(p) for p in self.permissions]
 
 
+class DeviceAuthMode(StrEnum):
+    SETUP = "setup"  # setup mode, any devices polling with an auth token that don't have one will save it
+    STRICT = "strict"  # all devices must have keys, and all keys must be set up with the API
+    LAX = "lax"  # devices may or may not use keys, but device with keys set must poll with them
```

Comment:
See my comment on the PR - `lax` seems pointless. What about my proposal of a different config structure?